report/src/main/webapp/bower.json

```diff
@@ -11,7 +11,10 @@
     "angular-ui-router": "^0.2.18",
     "angular-bootstrap": "^1.1.0",
     "angularAMD": "^0.2.1",
-    "lodash": "^4.11.0"
+    "lodash": "^4.11.0",
+    "FileSaver": "https://cdn.rawgit.com/eligrey/FileSaver.js/5733e40e5af936eb3f48554cf6a8a7075d71d18a/FileSaver.js",
```

Comment:
Why aren't we using standard approach for npm as for other libraries?:
* https://www.npmjs.com/package/canvas2pdf
* https://www.npmjs.com/package/file-saver

*canvas2pdf* was moved lately and provided link is not working... Also it would be better to specify a version of lib.